src/main/java/org/folio/service/impl/ItemLevelServicePointServiceImpl.java

```diff
@@ -0,0 +1,74 @@
+package org.folio.service.impl;
+
+import java.util.Collection;
+import java.util.Objects;
+import java.util.stream.Stream;
+
+import lombok.extern.log4j.Log4j2;
+
+import org.apache.commons.lang3.StringUtils;
+import org.folio.client.feign.CirculationClient;
+import org.folio.client.feign.SearchClient;
+import org.folio.domain.dto.AllowedServicePointsRequest;
+import org.folio.domain.dto.AllowedServicePointsResponse;
+import org.folio.repository.EcsTlrRepository;
+import org.folio.service.RequestService;
+import org.folio.service.UserService;
+import org.folio.spring.service.SystemUserScopedExecutionService;
+import org.springframework.stereotype.Service;
+
+@Log4j2
+@Service
+public class ItemLevelServicePointServiceImpl extends AllowedServicePointsServiceImpl {
+
+  public ItemLevelServicePointServiceImpl(SearchClient searchClient,
+    CirculationClient circulationClient, UserService userService,
+    SystemUserScopedExecutionService executionService, RequestService requestService,
+    EcsTlrRepository ecsTlrRepository) {
+
+    super(searchClient, circulationClient, userService, executionService,
+      requestService, ecsTlrRepository);
+  }
+
+  @Override
+  protected AllowedServicePointsResponse getForCreate(AllowedServicePointsRequest request) {
+    String instanceId = "";
```

Comment:
There is too much code duplication in your service implementations. Please try to extract just the parts that differ.